packages/ecr-credential-provider-1.30/ecr-credential-provider-1.30.spec

```diff
@@ -0,0 +1,46 @@
+%global goproject github.com/kubernetes
+%global gorepo cloud-provider-aws
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.29.0
+# %global gover 1.30.0
```

Comment:
nit: usually for macros it's not enough to comment them, you also have to escape them:
```suggestion
# %%global gover 1.30.0
```